packages/web-components/src/components/page-header/page-header.stories.ts

```diff
@@ -158,7 +223,17 @@ export const ContentWithHeroImage = {
     <cds-page-header>
       <div class="cds--css-grid">
         <div class="cds--sm:col-span-4 cds--md:col-span-4 cds--lg:col-span-8 cds--css-grid-column">
-          <cds-page-header-breadcrumb within-grid border=false>BreadcrumbBar</cds-page-header-breadcrumb>
+          <cds-page-header-breadcrumb ?border="${false}">
```

Comment:
```suggestion
          <cds-page-header-breadcrumb border=false>
```

Since `border` is by default true, even if it isn't set here, it will still render the border styles. Whenever attributes are set to true by default it's a bit tricky to shut off which is why I went with using `:host(#{$prefix}-page-header-breadcrumb[border='true']` in the styles